charts/tock/templates/admin_web.deployment.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: '{{ .Release.Name }}-admin-web'
+  labels:
+    app.kubernetes.io/name: admin-web
+    app.kubernetes.io/component: admin-web
+    app.kubernetes.io/version: {{ .Chart.Version }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.adminWeb.replicas }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: admin-web
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/resource: deployment
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: admin-web
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/resource: deployment
+    spec:
+      {{- include "adminWeb.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.adminWeb.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.adminWeb.podSecurityContext "enabled" | toYaml | nindent 10 }}
+      {{- end }}
+      initContainers:
+        - name: check-mongo
+          image:  {{ include "initContainer.image" . }}
+          {{- if .Values.global.initContainerImage.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.global.initContainerImage.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          command:
+            - sh
+            - -c
+            - |-
+              OK=0
+              echo "Checking mongo port"
+              while [ $OK != 1 ]; do
+                  echo -n "."
+                  nc -z {{ template "url.mongo.isup" . }} 27017 2>&1 >/dev/null && OK=1 || sleep 1
+              done
```

Comment:
The release deployment will be incomplete. We could have some cases where the service can't be up because resources are not enough on kube cluster ... until nodes are adder or pods be removed. So I think that waiting is not a real issue. Nevertheless you can uninstall the release if need.